include/annotated.hpp

```diff
@@ -153,6 +223,23 @@ struct Annotated {
         return *this;
     }
 
+    template<typename U>
+        requires std::is_same_v<std::remove_cvref_t<U>, T>
+    constexpr bool
+    validate(U &&value_) {
```

Comment:
I'd propose renaming to `validate_and_set` or similar.

Reply:
OK.